packages/core/helper-plugin/lib/src/utils/getYupInnerErrors/index.js

```diff
@@ -1,10 +1,11 @@
-import { get } from 'lodash';
-
 const getYupInnerErrors = (error) => {
-  return get(error, 'inner', []).reduce((acc, curr) => {
-    acc[curr.path.split('[').join('.').split(']').join('')] = {
-      id: curr.message,
-      defaultMessage: curr.message,
+  return (error?.inner || []).reduce((acc, currentError) => {
+    acc[currentError.path.split('[').join('.').split(']').join('')] = {
+      id: currentError.message,
+      defaultMessage: currentError.message,
+      values: Object.keys(currentError?.params || {})
+        .filter((key) => !['label', 'originalValue', 'path', 'value'].includes(key))
```

Comment:
This is the only part I don't really understand. Could you explain me how `currentError` looks like?

Reply:
I see. I'm not sure I like the approach of omitting a default set (value ...). Should we maybe pick the attributes we need for each error? In this case I'd say a `maxLength` could pick `max` from `params` only?

Reply:
Yes good idea. I've changed to code to pick the params based on the error type. WDYT?

Reply:
Yes, that looks good to me! 👍🏼

Reply:
Nice would you still like to include this in [here](https://github.com/strapi/strapi/pull/15692#issuecomment-1413961627)? Or merge this separately?

Reply:
I think we can merge your work separately, because it mostly affects the client-side error handling. The API doesn't give us the level of yup details at the moment I believe, so we need to talk to the BE first and see if they can expose that for us.